deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -9091,6 +9091,12 @@ spec:
             spec:
               description: ObjectZoneSpec represent the spec of an ObjectZone
               properties:
+                customEndpoints:
+                  description: 'If this zone cannot be accessed from other peer Ceph clusters via the ClusterIP Service endpoint created by Rook, you must set this to the externally reachable endpoint(s). You must also include the port in the definition. For example: "https://my-object-store.my-domain.net:443". In many cases, you should set this to the endpoint of the ingress resource that makes the CephObjectStore associated with this CephObjectStoreZone reachable to peer clusters. The list can have one or more endpoints pointing to different RGW servers and need to list all the endpoint(s) part of zone endpoint list in multisite configuration'
```

Comment:
the doc says that not all endpoints available i a zone need to be listed. but this description says that all endpoints need to be listed.